blocks/product-details/requisition-list.js

```diff
@@ -21,49 +21,34 @@
  * rendering the requisition list component, event handling, and notifications.
  */
 
-import {
-  InLineAlert,
-  Icon,
-  provider as UI,
-} from '@dropins/tools/components.js';
+import { Icon, InLineAlert, provider as UI } from '@dropins/tools/components.js';
 import { h } from '@dropins/tools/preact.js';
 import { events } from '@dropins/tools/event-bus.js';
-import * as rlApi from '@dropins/storefront-requisition-list/api.js';
 import { render as rlRenderer } from '@dropins/storefront-requisition-list/render.js';
-import { RequisitionListSelector } from '@dropins/storefront-requisition-list/containers/RequisitionListSelector.js';
-import { companyEnabled } from '@dropins/storefront-company-management/api.js';
+import {
+  RequisitionListSelector,
+} from '@dropins/storefront-requisition-list/containers/RequisitionListSelector.js';
 import * as pdpApi from '@dropins/storefront-pdp/api.js';
 import { checkIsAuthenticated } from '../../scripts/commerce.js';
 
 // Initialize dropins
 import '../../scripts/initializers/company.js';
 import '../../scripts/initializers/requisition-list.js';
 
-/**
- * Checks if requisition list feature is fully enabled
- * Requires both B2B (company) and requisition list to be enabled
- * @returns {Promise<boolean>} True if feature is enabled
- */
-export async function isRequisitionListFeatureEnabled() {
-  try {
-    const isB2BEnabled = await companyEnabled();
-    const isRequisitionListEnabled = await rlApi.isRequisitionListEnabled();
-    return isB2BEnabled && isRequisitionListEnabled;
-  } catch (error) {
-    console.error('Error checking requisition list feature status:', error);
-    return false;
-  }
-}
```

Comment:
So are we doing both checks (B2B and reqList) in dropin?

Reply:
Yup, no need to check it again here. with just one check is enough